### DISCUSSION TITLE
Add Elasticsearch datasource in Jaeger trace to logs

### DISCUSSION THREAD

**Discussion by polgeko** [grafana/grafana#48770] [Observability]
Hi, I would like to know when will you add a feature in Jaeger datasource who lets you correlate trace to logs with Elasticsearch Backend.
I know that now is only available with Loki and Splunk. Do you know when will you add Elasticsearch support?

Thanks !! Keep working like you are :)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
